smart-py/objkt_swap.py

```diff
@@ -17,135 +38,336 @@ def __init__(self, objkt, hdao, manager, metadata, curate):
             genesis = sp.timestamp(0),
             curate = curate,
             locked = False
-            )
-    
+        )
+
     @sp.entry_point
     def genesis(self):
-        sp.verify((sp.sender == self.data.manager) & ~(self.data.locked))
+        sp.verify(
+            (
+                sp.sender == self.data.manager
+            ) & ~(
+                self.data.locked
```

Comment:
Explain the use of the lock.

Reply:
so that genesis + 45 days may only be called once

Reply:
Could you add that as a comment?